scraper.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
```

Comment:
ref comment above: "consider an "installing dependencies" piece, either here or in the code with an inline comment about using pip3 (or another way?)"